content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -29,7 +29,7 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/created-by=controller-manager`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```

Comment:
Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).
```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```